### DISCUSSION TITLE
tutorials/intro-strawberry/05-apollo-explorer

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#173] [course feedback]
# tutorials/intro-strawberry/05-apollo-explorer

Learn how to build and run queries using Apollo Sandbox Explorer, a GraphQL IDE.

https://www.apollographql.com/tutorials/intro-strawberry/05-apollo-explorer

**Comment by robwma** [comment]
In the beginning of this text it says to navigate to localhost:8000 but as a newbie to Apollo I am not sure how to start whatever will be behind the localhost:8000, am I suppose to run the main.py  or install something else?  If I click on the Apollo GraphOS which one hybrid or cloud do I need to use?

**Reply by mabuyo** [reply]
Hi there @robwma ! [In lesson 1](https://www.apollographql.com/tutorials/intro-strawberry/01-overview-setup), we set up the project and ran the server using the command `uvicorn main:app --reload`. That starts up the server on localhost:8000 and you can go from there.

Can you clarify what you mean by "If I click on the Apollo GraphOS"?

**Reply by robwma** [reply]
D'oh! I forgot that I had closed down the uvicorn.. Before you mentioned this, I thought I had to download or use Apollo GraphOS to get this to work.

**Reply by mabuyo** [reply]
Totally valid, it was a few lessons ago. We'll add a note at the top to remind folks to start up their server before heading over to localhost:8000. Thanks for providing feedback!
